towerlib/entities/job.py

```diff
@@ -726,6 +806,56 @@ def cancel(self):
         response = self._tower.session.post(url)
         return response.ok
 
+    def _fetch(self, printed_nodes):
```

Comment:
I find the use of the underscore in the variables of this method a bit distructing and confusing. I am not sure that all of them need that, i guess you are trying to save from shadowing either build ins or external variables but it does make it hard to read. Maybe a bit more targeted name would be better? If `_id` was `job_id` and everything else missed the underscore i guess it would still be fine and it would be more readable.

Reply:
This has been updated